components/status_dropdown/index.jsx

```diff
@@ -21,11 +25,15 @@ function mapStateToProps(state) {
     }
 
     const userId = currentUser.id;
+    const ctz = getUserTimezone(state, userId);
+    const etz = areTimezonesEnabledAndSupported(state);
     return {
         userId,
         profilePicture: Client4.getProfilePictureUrl(userId, currentUser.last_picture_update),
         autoResetPref: get(state, Preferences.CATEGORY_AUTO_RESET_MANUAL_STATUS, userId, ''),
         status: getStatusForUserId(state, userId),
+        userTimezone: ctz,
+        enableTimezone: etz,
     };
 }
```

Comment:
there is no need to instantiate variables for the values.

They can immediately saved to the corresponding properties:

```suggestion
    return {
        userId: currentUser.id,
        profilePicture: Client4.getProfilePictureUrl(userId, currentUser.last_picture_update),
        autoResetPref: get(state, Preferences.CATEGORY_AUTO_RESET_MANUAL_STATUS, userId, ''),
        status: getStatusForUserId(state, userId),
        userTimezone: getUserTimezone(state, userId),
        enableTimezone: areTimezonesEnabledAndSupported(state),
    };
}
```